Assets/Scripts/GpsSensor.cs

```diff
@@ -59,14 +61,16 @@ private void ReportPosition()
         // z+ is north, x+ is east
         double[] GPS = Utilities.metersToGPS(new double[] {
             transform.position.z + _noise * Utilities.GaussianRandom(),
-            transform.position.x + _noise * Utilities.GaussianRandom()},
-            new double[] {initLat, initLon});
+            transform.position.x + _noise * Utilities.GaussianRandom(),
+            transform.position.y + _noise * Utilities.GaussianRandom()},
```

Comment:
If we are reporting the altitude in meters anyway, I don't think we need to pass it into this metersToGPS function